manifests/init.pp

```diff
@@ -113,6 +113,10 @@
   # incase the jira service is managed outside of puppet. eg: using the
   # puppetlabs-corosync module: 'crm resource stop jira && sleep 15'
   $stop_jira = 'service jira stop && sleep 15',
+  # Whether to manage the 'check-java.sh' script, and where to retrieve
+  # the script from.
+  $script_check_java_manage = false,
+  $script_check_java_template = undef,
```

Comment:
I don't believe using `'template(...)'` as a string will work, as _template_ is a function, and the `content` attributes of files doesn't parse the string. So this either has be set to `file` or `template` in the content attribute of the file resource, and I opted for `template` as that is what you originally requested.

Given the file doesn't use any variables, it doesn't make much sense, but at least it seems consistent with the rest of the module?